### PR TITLE
[ADF-3889] [Start Process Cloud] - APS2 - error displayed when try to  create a process with a name bigger the 255 characters

### PR DESCRIPTION
**Pull request by sivakumar414ram**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://issues.alfresco.com/jira/browse/ADF-3889

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
                     matInput
                     formControlName="processInstanceName" 
                     id="processName">
-                <mat-error *ngIf="processInstanceName.hasError('required')">
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html]: Why we are removing this validation?

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html [hunk 2]
```diff
@@ -15,8 +15,8 @@
...
-                    {{ 'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.ERROR.PROCESS_NAME_REQUIRED' | translate }}
+                <mat-error id="adf-process-name-error-message" *ngIf="errorMessage">
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html]: can we do the same as start task?
```
<mat-error *ngIf="nameController.hasError('maxlength')">
                        {{ 'ADF_CLOUD_START_TASK.ERROR.MAXIMUM_LENGTH' | translate : { characters : maxNameLength } }}
                    </mat-error>
```

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts [hunk 3]
```diff
@@ -36,6 +36,8 @@ import { ProcessDefinitionCloud } from '../models/process-definition-cloud.model
 })
 export class StartProcessCloudComponent implements OnChanges, OnInit {
 
+    MAX_LENGTH: number = 255;
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts]: use the same convention as start task
    static MAX_NAME_LENGTH = 255;

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts [hunk 4]
```diff
@@ -76,18 +78,23 @@ export class StartProcessCloudComponent implements OnChanges, OnInit {
     processForm: FormGroup;
     processPayloadCloud = new ProcessPayloadCloud();
     filteredProcesses: ProcessDefinitionCloud[] = [];
+    maxProcessInstanceNameLength: number = this.MAX_LENGTH;
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts]: /** Maximum length of the task name. */
    @Input()
    maxNameLength: number = StartTaskCloudComponent.MAX_NAME_LENGTH;

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts [hunk 5]
```diff
@@ -76,18 +78,23 @@ export class StartProcessCloudComponent implements OnChanges, OnInit {
...
     isLoading = false;
+    errorMessage: string;
+    private validationMessages = { required: 'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.ERROR.PROCESS_NAME_REQUIRED',
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts]: what is that ?

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts [hunk 6]
```diff
@@ -76,18 +78,23 @@ export class StartProcessCloudComponent implements OnChanges, OnInit {
...
     }
 
     ngOnInit() {
         this.processForm = this.formBuilder.group({
-            processInstanceName: new FormControl(this.name, Validators.required),
+            processInstanceName: new FormControl(this.name, [Validators.required, Validators.maxLength(this.maxProcessInstanceNameLength)]),
             processDefinition: new FormControl('', [Validators.required, this.processDefinitionNameValidator()])
         });
 
+        this.setProcessNameValidationMessage();
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts]: why?

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html (hunk 2 above)
Comment (sivakumar414ram) [reply lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html]: ok

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts (hunk 3 above)
Comment (sivakumar414ram) [reply lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts]: ok

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts (hunk 4 above)
Comment (sivakumar414ram) [reply lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts]: fixed

**Pull request merged**